clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+import {Request} from '../lib/request';
+import {Response} from '../lib/response';
+import {AWSError} from '../lib/error';
+import {Service} from '../lib/service';
+declare class ACM extends Service {
+  /**
```

Comment:
Well, for S3, since you currently have `constructor(options?: ServiceConfigurationOptions & UseDualstackConfigOptions)`, couldn't that just be changed to `constructor(options?: ServiceConfigurationOptions & UseDualstackConfigOptions & ServiceParams)`?